pwiz_tools/Skyline/TestTutorial/TargetedMSMSTutorialTest.cs

```diff
@@ -1133,7 +1133,7 @@ private void TestPropertySheet()
             Assert.IsNotNull(propertyGrid);
 
             // Checks the number of properties displayed is the expected number
-            int expectedPropCount = isSmallMolecules ? 7 : 11;
+            int expectedPropCount = isSmallMolecules ? 6 : 10;

```

Comment:
Why did this change?

Reply:
Retention time property was missing in action. I fixed it now.